zeek/template/84123640-bf99-415d-b69c-de438c46027e.query

```diff
@@ -1 +1,3 @@
-tag=zeekconn ax resp==%%IP%% orig resp_bytes != - | stats unique_count(orig) as "Active Clients" | numbercard "Active Clients"
\ No newline at end of file
+tag=zeekconn ax "id.resp_h == "%%IP%%" "id.orig_h" resp_bytes != "-"
```

Comment:
you're missing a closing " after id.resp_h

Reply:
Corrected